WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -485,7 +485,11 @@ private async Task CreateBlameRoundAsync(Round round, CancellationToken cancella
 			.Where(x => !Prison.IsBanned(x, Config.GetDoSConfiguration(), DateTimeOffset.UtcNow))
 			.ToHashSet();
 
-		RoundParameters parameters = RoundParameterFactory.CreateBlameRoundParameter(feeRate, round);
+		RoundParameters parameters = RoundParameterFactory.CreateBlameRoundParameter(feeRate, round) with
+		{
+			MinInputCountByRound = Config.MinInputCountByBlameRound
+		};
```

Comment:
Suggestion:
We could move this `with { ... }` part inside the `roundParameterFactory.CreateBlameRoundParameter` function.

It would look like this:

```cs
public virtual RoundParameters CreateBlameRoundParameter(FeeRate feeRate, Round blameOf) =>
	RoundParameters.Create(
		Config,
		Network,
		feeRate,
		Config.CoordinationFeeRate,
		blameOf.Parameters.MaxSuggestedAmount) with
	{
		MinInputCountByRound = Config.MinInputCountByBlameRound
	};
```

Not sure if it's worth it or not, so it's your call.

Reply:
You probably did, but I didn't understand it I guess.

Anyways, this is nit and both solutions work.